tests/foreman/ui/test_activationkey.py

```diff
@@ -422,7 +422,6 @@ def test_positive_update_cv(session, module_org, cv2_name, target_sat):
 
 
 @pytest.mark.run_in_one_thread
-@pytest.mark.skip_if_not_set('fake_manifest')
```

Comment:
We can ultimately change this to verify if manifest and manifester settings files availability.

Reply:
@synkd Any comments on this ?

Reply:
@jyejare I assume you're talking about creating a new marker to skip tests if manifester's conf requirements are not in place? If so, that sounds reasonable, but let's leave that work for a separate PR.

Reply:
Another thought on this, reworking the `fake_manifest` marker would be a good opportunity to mark all tests that use manifester to make it easier to collect those tests as group.

Reply:
+1

Reply:
Created a new issue for this and assigned it to you @synkd !